index.html

```diff
@@ -127,15 +129,26 @@ <h2 class="section-title">Add New Task</h2>
               </div>
               <label class="label" for="time">Time:</label>
               <input class="input-time" type="time" id="time" name="time" />
-              <button class="submit-button" name="submit-button" type="button">
+              <button
+                class="submit-button"
+                name="submit-button"
+                type="button"
+                onclick="addNewTask()"
+              >
                 Save
               </button>
-              <button class="delete-button" name="delete-button" type="button">
+              <button
+                id="edit-task-button"
+                class="delete-button"
+                name="delete-button"
+                type="button"
+              >
```

Comment:
Please undo everything related to edit task.

Reply:
from this message I get everything related to "Add new task" must be in this state so I deleted the id="edit-task-button" from button tag and the header tag related to edit task